uxarray/grid/neighbors.py

```diff
@@ -345,24 +345,26 @@ def query_radius(
                 coords, r, return_distance, count_only, sort_results
             )
 
-            ind = np.asarray(ind[0], dtype=INT_DTYPE)
+            ind = [np.asarray(cur_ind, dtype=INT_DTYPE) for cur_ind in ind]
```

Comment:
Do we need to convert to integer? sklearn's KDTree.query_radius already returns integers

Reply:
`INT_DTYPE` is our standard data type used for indexing, its for consistency.

Reply:
seems like a lot of extra typing for consistency.